README.md

```diff
@@ -51,19 +55,35 @@ os.environ['SNOWFLAKE_ROLE'] = '<your-snowflake-role>'
 os.environ['SNOWFLAKE_WAREHOUSE'] = '<your-snowflake-warehouse>'
 os.environ['SNOWFLAKE_USER'] = '<your-snowflake-user>'
 os.environ['SNOWFLAKE_PASSWORD'] = '<your-snowflake-password>'
+os.environ['SNOWFLAKE_ACCOUNT_LOCATOR']='<your-snowflake-account-locator>'
 os.environ['SNOWFLAKE_HOST'] = '<your-snowflake-host>'
 ```
+
+## admin-app
+
+### Generation and validation using admin app
+
+1. Ensure you have installed the Python package. Note, the version below should be the latest version under the `dist/` directory.
```

Comment:
remove this 

 Note, the version below should be the latest version under the `dist/` directory.